src/content/en/updates/2017/03/performant-expand-and-collapse.md

```diff
@@ -0,0 +1,294 @@
+      animation-name: menuContentsAnimation;
+      animation-duration: 0.2s;
+      animation-timing-function: linear;
+    }
+
+This causes the animations to run that were created in the previous step. Because the baked
+animations are already eased, the timing function needs to be set to `linear` otherwise you’ll
+ease between each keyframe which will look very weird!
+
+When it comes to collapsing the element back down there are two options: update the CSS animation
+to run in reverse rather than forwards. This will work just fine, but the "feel" of the animation
+will be reversed, so if you used an ease-out curve the reverse will feel eased *in*, which will
+make it feel sluggish. A more appropriate solution is to create a *second pair* of animations for
+collapsing the element. These can be created in exactly the same way as the expand keyframe
+animations, but with swapped start and end values.
+
+    const xScale = 1 + (x - 1) * easedStep;
+    const yScale = 1 + (y - 1) * easedStep;
+
+## A more advanced version: circular reveals
+
+It’s also possible to use this technique to make circular expand and collapse animations.
+
+<div style="display: flex; justify-content: center">
+  <video src="/web/updates/images/2017/03/performant-expand-and-collapse/circular.mp4"
+      loop autoplay></video>
+</div>
+
+The principles are largely the same as the previous version, where you scale an element, and
+counter-scale its immediate children. In this case the element that’s scaling up has
+a `border-radius` of 50%, making it circular, and is wrapped by *another* element that
+has `overflow: hidden`, meaning that you don’t see the circle expand outside of the element bounds. 
+
+A word of warning on this particular variant: Chrome has blurry text on low DPI screens during the
+animation because of rounding errors due to the scale and counter-scale of the text. If you’re
+interested in the details for that
+[there’s a bug filed that you can star and follow](https://crbug.com/704067).
+
+The code for the circular expand effect can be found in
+[the GitHub repo](https://github.com/GoogleChrome/ui-element-samples/tree/gh-pages/animated-clip/advanced).
+
+## Conclusions
+
+So there you have it, a way to do performant clip animations using scale transforms. In a perfect
+world it would be great to see clip animations be accelerated (there’s
+[a Chromium bug for that](https://bugs.chromium.org/p/chromium/issues/detail?id=686074) made by 
+Jake Archibald), but until we get there you should be cautious when animating clip or clip-path,
+and definitely avoid animating width or height. 
+
+It would also be handy to use Web Animations for effects like this, because they have a JavaScript
+API but can run on the compositor thread if you only animate transforms and opacity. 
+Unfortunately [support for Web Animations isn’t great](http://caniuse.com/#feat=web-animation), 
+though you could use Progressive Enhancement to use them if they’re available.
```

Comment:
Change 'Progressive Enhancement' to 'progressive enhancement'. It's not a proper noun.